en/docs/learn/authentication-session-persistence.md

```diff
@@ -107,10 +107,11 @@ few tasks to clean the data in the database for some given conditions.
 
 #### Task Pool
 
-We are storing above data within a critical path which controls the
-authentication. So we have introduced a pooling mechanism to put the
+We employed task pool on some older releases, when storing above data within a critical path which controls the
+authentication. Here we have introduced a pooling mechanism to put the
 data persistence tasks into that pool and continue the critical path
-without blocking it.
+without blocking it. However this has not proven not providing expected results, and caused some unwanted side effects. Hence we later reverted our recommendation not to use this feature.
```

Comment:
Shall we restructure this sentence to make it clearer? 

However, this did not prove to provide the expected results. It further caused a few side effects. Hence, it was later reverted and it is not recommended to use this feature anymore.